packages/@aws-cdk/aws-stepfunctions-tasks/README.md

```diff
@@ -327,31 +329,60 @@ Step Functions supports [ECS/Fargate](https://docs.aws.amazon.com/step-functions
 
 [RunTask](https://docs.aws.amazon.com/step-functions/latest/dg/connect-ecs.html) starts a new task using the specified task definition.
 
+#### EC2
+
+The following example runs a job from a task definition on EC2
+
 ```ts
-import * as ecs from '@aws-cdk/aws-ecs';
```

Comment:
Perhaps keep this line to clarify that the `ecs.` prefixed constructs are coming from the `aws-ecs` module.

Reply:
good catch, just a miss on my part. I'd done it for Fargate and not for EC2. updated!